src/features/landing/components/AboutVilnaSection/AboutVilnaBackgroundWithMainSvg/AboutVilnaBackgroundWithMainSvg.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from '@mui/material';
+import React from 'react';
+
+interface IBackgroundProps {
+  children?: React.ReactNode;
+  style?: React.CSSProperties;
+}
+
+const styles = {
+  mainBox: {
+    backgroundImage: `url("/assets/svg/about-vilna/background_main.svg")`,
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    height: '100%',
+    width: '100%',
+    maxWidth: '49.875rem', // 798px
```

Comment:
remove comments, this comment does not carry any information
read this https://refactoring.guru/ru/smells/comments